src/buffer/out/UTextAdapter.h

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+// Can't forward declare the UErrorCode enum. Thanks C++.

```

Comment:
huh what does it do?

Reply:
Checking it again, it's just a warning:

> UTextAdapter.h(6,6): warning C4471: 'UErrorCode': a forward declaration of an unscoped enumeration must have an underlying type

https://learn.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-4-c4471

I'll remove the comment, because including icu.h there is fine. The file is now only used in .cpp files anyways.
(I don't quite get how the warning is useful at all, since the compiler can just recognize the error when it finds the actual enum definition in a .cpp file. I don't get it...)